src/main/java/jdbchomework/dao/hibernate/CompaniesHibDao.java

```diff
@@ -0,0 +1,55 @@
+package jdbchomework.dao.hibernate;
+
+import jdbchomework.dao.model.CompaniesDao;
+import jdbchomework.entity.Company;
+import jdbchomework.entity.Developer;
+import org.hibernate.HibernateException;
+import org.hibernate.query.NativeQuery;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.Session;
+import org.slf4j.LoggerFactory;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CompaniesHibDao extends AbstractHibDao<Company> implements CompaniesDao {
+    private org.slf4j.Logger log = LoggerFactory.getLogger(CompaniesHibDao.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "log" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)